### PR TITLE
Remove `EditPageLayout`

### PR DESCRIPTION
**Pull request by Flips2001**
---

Removed EditPageLayout from the library and all its instances

## PR Checklist

-   [x] Verify if the change requires a changeset. See [CONTRIBUTING.md](https://github.com/vivid-planet/comet/blob/HEAD/CONTRIBUTING.md)
-   [x] Link to the respective task if one exists: COM-636
-   [x] Provide screenshots/screencasts if the change contains visual changes

<details>
    <summary>Screenshots/screencasts</summary>
    
</details>

### CODE REVIEW

File: .changeset/many-oranges-mate.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-admin": major
+---
+
+`Deleted EditPageLayout`
```
Comment (jamesricky) [inline .changeset/many-oranges-mate.md:5]:
```suggestion
Remove `EditPageLayout`
```

**Review by jamesricky — DISMISSED** [review]

**Review by johnnyomair — COMMENTED** [review]
Please provide screenshots for this change. I can't believe that this doesn't break things.

File: .changeset/many-oranges-mate.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-admin": major
+---
+
+Remove `EditPageLayout`
```
Comment (johnnyomair) [inline .changeset/many-oranges-mate.md:5]: Please extend this changeset why this isn't necessary anymore and what devs should do instead.

**Comment by johnnyomair** [comment]
@thomasdax98 could you take this on after #2051 has been merged?

**Review by jamesricky — APPROVED** [review]

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**